lib/Page.js

```diff
@@ -544,6 +544,13 @@ class Page extends EventEmitter {
   }
 
   /**
+  * @return {!Promise<?Response>}
```

Comment:
Please remove the "@return" statement - it's not needed since the function doesn't return anything. (technically, the function returns a promise, but there's an agreement to not declare empty promises)